www/templates/control/main-control.html

```diff
@@ -10,60 +10,41 @@
-      <div ng-click="forceSync()" ng-style="getIconButtonStyle('D6C780')"><i class="ion-android-sync" ng-style="getIconStyle()"></i></div>
-    </div>   
-    <div class="control-list-item">
-      <div class="control-list-text">DEVELOPER ZONE</div>
+    
+    <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+      <div style="padding: 15px 15px; float: left;">Developer zone</div>
       <div ng-click="toggleCollection()" ng-style="getIconButtonStyle('D6C780')"><i ng-class="getExpandButtonClass()" ng-style="getIconStyle()" id="expandButton"></i></div>
     </div>
     
     
     <div class="item-accordion" ng-show="collectionExpanded()">
-      <div class="control-list-item">
-        <div class="control-list-text">Refresh</div>
-        <div ng-click="refreshScreen()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-loop" ng-style="getIconStyle()"></i></div>
-      </div>
-      <div class="control-list-item">
-        <div class="control-list-text">{{parseState(settings.collect.state)}}</div>
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">{{parseState(settings.collect.state)}}</div>
         <div ng-click="forceState()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-edit" ng-style="getIconStyle()"></i></div>
       </div>
-      <div class="control-list-item">
-        <div class="control-list-text">Check log</div>
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">Check log</div>
         <div ng-click="showLog()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-ios-arrow-right" ng-style="getIconStyle()"></i></div>
       </div>      
-      <div class="control-list-item">
-        <div class="control-list-text">Check sensed data</div>
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">Check sensed data</div>
         <div ng-click="showSensed()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-ios-arrow-right" ng-style="getIconStyle()"></i></div>
       </div>   
-      <div class="control-list-item">
-        <div class="control-list-text">Check map</div>
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">Check map</div>
         <div ng-click="showMap()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-ios-arrow-right" ng-style="getIconStyle()"></i></div>
       </div>   
-      <div class="control-list-item">
-        <div class="control-list-text">Collection</div>
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">Collection</div>
         <div ng-click="editConfig($event)" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-edit" ng-style="getIconStyle()"></i></div>
```

Comment:
Note that this was changed to `editCollectionConfig` as part of my changes. You copied the code over so you need to change this as well.